homeassistant/components/nasweb/strings.json

```diff
@@ -0,0 +1,46 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add NASweb device",
+        "description": "{nasweb_schema_img}NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions and configured actions.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "ha_address": "Home Assistant Address"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "missing_status": "Did not receive status from NASweb, check logs",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "issues": {
+    "invalid_authentication": {
+      "title": "[%key:common::config_flow::error::invalid_auth%]",
+      "description": "{domain_name}[{device_name}]: Invalid username/password. Most likely user changed password or was removed. Delete this entry and create new one with correct username/password."
+    },
```

Comment:
Do you plan to implement a reauth flow?

Reply:
Yes, I do plan to implement it. From what I understand, I shouldn't include too much in the first PR. Should the reauth flow be included in the first PR, or should I leave it for future PRs?

Reply:
It's fine to leave it for a future PR 👍